classes/class-wc-connect-taxjar-integration.php

```diff
@@ -945,10 +995,6 @@ public function calculate_tax( $options = array() ) {
 				'to_city'    => $to_city,
 			);
 
-			if ( 'GB' === $to_country ) {
-				$location['to_state'] = '';
-			}
-
```

Comment:
The `GB` check was for https://github.com/Automattic/woocommerce-services/pull/1899. If we moved it, we should probably do some regression test on the issue.

Reply:
I moved it here to colocate colocate all the "state"/"country" checks: https://github.com/Automattic/woocommerce-services/blob/624b44e509c163715364e2bf2883f3e3f2fd787f/classes/class-wc-connect-taxjar-integration.php#L1049

I checked the scenarios in that PR as well, fell free to double check 👍